CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 - No changes yet.
 
+## [1.21.0] - 2019-07-23
+### Fixed
+- Field compilation is now case-sensitive.
```

Comment:
This should reflect how it affects the user.

```suggestion
- Fixed field compilation to allow fields with similar looking names and
  different casing.
```

Reply:
👍